src/Bicep.Core.Samples/Files/ResourcesTenant_CRLF/main.bicep

```diff
@@ -43,4 +43,9 @@ resource yetAnotherSet 'Microsoft.Management/managementGroups@2020-05-01' = [for
   dependsOn: [
     anotherSet[0]
   ]
+}]
+
+output managementGroupIds array = [for i in range(0, length(managementGroups)): {
+  name: yetAnotherSet[i].name
+  displayName: yetAnotherSet[i].properties.displayName

```

Comment:
>[i] [](start = 28, length = 3)

Supporting direct references to collections will make this a lot less awkward. It'll come in a different PR, though.